### PR TITLE
feat: add "Powered by MagicKBase" text

### PR DESCRIPTION
**Pull request by PainterPuppets**
resolved: https://github.com/Magickbase/ckb-explorer-public-issues/issues/677

### CODE REVIEW

**Comment by Keith-CY** [comment]
Hold on, I found the alignment is a bit rare, and I will check it with the designer
image

**Comment by Keith-CY** [comment]
> Hold on, I found the alignment is a bit rare, and I will check it with the designer image

The design has been updated, be noted that `left/right ends` are aligned to the footer but not the body.

**Comment by Keith-CY** [comment]
> > Hold on, I found the alignment is a bit rare, and I will check it with the designer image
> 
> The design has been updated, be noted that `left/right ends` are aligned to the footer but not the body.

Please check @PainterPuppets

**Comment by PainterPuppets** [comment]
image
 
optimized

**Comment by Keith-CY** [comment]
I found the unlimited expanded footer is a bit weird compared to the current one. How do you thing @Sven-TBD

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -132,33 +132,40 @@ export default memo(() => {
...
+            {Footers[2].items.map(item => (
+              <FooterImageItem item={item} key={item.label} />
+            ))}
+          </div>
+        </FooterMenuPanel>
+      </div>
+      <div style={{ width: '100%' }}>
+        <div className="footerCopyright container">
+          <a className="power" href="https://www.magickbase.com" target="_blank" rel="noreferrer">
+            Powered by MagicKBase
```
Comment (Keith-CY) [inline src/components/Footer/index.tsx:164]:
```suggestion
            Powered by Magickbase
```

**Comment by FrederLu** [comment]
UI：
image

image

1、Comparing the UI draft, we found that the color of `Powered by MagicKBase` was `#999999` and now it is `#FFFFFF`.
2、`Powered by MagicKBase` needs to jump, but the jump icon is missing next to the text. At the same time, the system color is missing when the mouse hovers.

image

3、The page display after clicking the jump is abnormal.

**Comment by Kirl70** [comment]
> UI： image
> 
> image
> 1、Comparing the UI draft, we found that the color of `Powered by MagicKBase` was `#999999` and now it is `#FFFFFF`. 2、`Powered by MagicKBase` needs to jump, but the jump icon is missing next to the text. At the same time, the system color is missing when the mouse hovers.
> 
> image
> 3、The page display after clicking the jump is abnormal.

Yes, because the previous requirements did not specify the need to link to the Magickbase website, an external link icon was added and the text color was changed. Additionally, a horizontal line was added above the text to achieve visual balance.

image

**Review by FrederLu — APPROVED** [review]

**Pull request merged**